docs/.eleventy.js

```diff
@@ -316,6 +316,10 @@ module.exports = function(eleventyConfig) {
         "./node_modules/algoliasearch/dist/algoliasearch-lite.esm.browser.js": "/assets/js/algoliasearch.js"
     });
 
+    eleventyConfig.addPassthroughCopy({
+        "./src/developer-guide/**/*.svg": "/assets/images"
```

Comment:
Should we do this to make sure for images other than this one?
```suggestion
        "./src/**/**/*.svg": "/assets/images"
```

Reply:
Or maybe just move these image to the central folder,  `/assets/images`? @mdjermanovic what do you think?

Reply:
> Or maybe just move these image to the central folder, `/assets/images`? @mdjermanovic what do you think?

Yeah, it might be best to just move them to `/assets/images`. We could then make subdirectories `/assets/images/architecture` and `/assets/images/code-path-analysis`.

Reply:
done